doc/source/whatsnew/v0.24.0.txt

```diff
@@ -629,7 +629,7 @@ Datetimelike
 - Bug in :meth:`DataFrame.eq` comparison against ``NaT`` incorrectly returning ``True`` or ``NaN`` (:issue:`15697`, :issue:`22163`)
 - Bug in :class:`DatetimeIndex` subtraction that incorrectly failed to raise ``OverflowError`` (:issue:`22492`, :issue:`22508`)
 - Bug in :class:`DatetimeIndex` incorrectly allowing indexing with ``Timedelta`` object (:issue:`20464`)
--
+- Bug in :class:`DatetimeIndex` where addition of Week offset with weekday=6 had no effect (:issue:`22465`)
```

Comment:
Was this bug present in 0.23.x? I think it wasn't, in which case this entry can be removed as users never saw it.

Reply:
No I don't think this bug was in 0.23.x because the changes causing this bug are also part of 0.24.0 so I guess we can remove this whatsnew entry

Reply:
Tests are passing.  Don't forget to do this!  Either @reidy-p or @TomAugspurger